cli/app/app.go

```diff
@@ -365,3 +393,19 @@ func sortServicesFirst(objs *[]runtime.Object) {
 	ret = append(ret, others...)
 	*objs = ret
 }
+
+func askForConfirmation() bool {
+	var response string
+	_, err := fmt.Scanln(&response)
+	if err != nil {
+		logrus.Fatal(err)
+	}
+	if response == "yes" {
+		return true
+	} else if response == "no" {
+		return false
+	} else {
+		fmt.Println("Please type yes or no and then press enter:")
```

Comment:
I'm thinking we can also print everything that's going to be deleted. Something like "Here are the resources to be deleted. Are you sure (yes/NO)?"